Firestore/Protos/nanopb/google/type/latlng.nanopb.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Google LLC
+ * Copyright 2022 Google LLC
```

Comment:
Why only the years under Protos changed, but not the others?

Reply:
For script generating nanopb files, do we need to run it manually every time when changing the codebase?

Reply:
The copyright date should be the first year a file is published externally. It's ok to update after, but not necessary.